service/repository/test_file_repository.py

```
@@ -0,0 +1,157 @@
+import os
+import unittest
+
+from service.repository.file_repository import FileRepository
+
+class MockModel:
+    uid: int
+    model_content: str
+
+class TestFileRepository(unittest.TestCase):
+
+    def __prepare_mock_object(self):
```

Ruff

Missing return type annotation for private function `__prepare_mock_object`

(ANN202)